### PR TITLE
set typing for arguments of functions

### PR DESCRIPTION
**Pull request by masoudgs**
I've made this PR to fix `tslint` errors.

Changes:
- Add typings to `renderIcon`, `renderPreviewKind`, `getIcon`

### CODE REVIEW

File: template/icon.nunjucks [hunk 1]
```diff
@@ -14,7 +14,7 @@ export interface IconProps {
     color?: string
     className?: string
     height?: number
-    kind?: string
+    kind?: string | any
```
Comment (jackple) [inline template/icon.nunjucks:17]: when to use `any`?

Comment (masoudgs) [reply template/icon.nunjucks:17]: I figured out that in some cases like `<Icon preview />` we don't need to `kind` attr which in this case an `undefined` situation for the `kind` props in the render function will occur.
https://github.com/jackple/typescript-react-svg-icon-generator/blob/e86d80c06e6fe456c5284866e1caaf65d4cdf31f/template/icon.nunjucks#L35

Comment (jackple) [reply template/icon.nunjucks:17]: so, `<Icon preview />` is fine, why `any` ? I still do not understand.

Comment (jackple) [reply template/icon.nunjucks:17]: As you said, you don't need `kind` attr

Comment (masoudgs) [reply template/icon.nunjucks:17]: While you make an interface property optional like `kind?: string`, the optional property make an `undefined` value which typescript needs to know what exact value will assign to this property.
So here `any` type will act just for getting rid `tslint` warning.

Comment (masoudgs) [reply template/icon.nunjucks:17]: i suggest you read about `strict null checks` of TypeScript [here](https://basarat.gitbooks.io/typescript/docs/options/strictNullChecks.html)

Comment (jackple) [reply template/icon.nunjucks:17]: OK, I have merged it. And thank you!

Comment (jackple) [reply template/icon.nunjucks:17]: latest version is 1.1.9

**Pull request merged**